packages/@lightningjs/ui-components/src/components/CardContent/CardContentVerticalSmall.stories.js

```diff
@@ -50,21 +49,24 @@ export const CardContentVerticalSmall = args =>
 
 CardContentVerticalSmall.storyName = 'CardContentVerticalSmall';
 
+//Creating a shallow copy of CardContent.args object and removing metadata details property from cardContentArgs
+const cardContentArgs = { ...CardContent.args };
+delete cardContentArgs.metadata_details;
+
 CardContentVerticalSmall.args = {
   mode: 'focused',
   ...CardContent.args
```

Comment:
I believe this should be `cardContentArgs` instead of `CardContent.args`?

Reply:
Oh yeah, you are right. I didn't assign the new copy.